week-3/extended-project/nowhere/views/about.erb

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>NoWhere | About us</title>
+    <link
+      rel="icon"
+      type="image/png"
+      href="https://i.imgur.com/3t8FYL1.png"

```

Comment:
After downloading a running this project, the image indicated on line 27 is not displayed in the browser.